clang/lib/AST/ASTContext.cpp

```diff
@@ -14296,6 +14296,30 @@ void ASTContext::getFunctionFeatureMap(llvm::StringMap<bool> &FeatureMap,
   }
 }
 
+static SYCLKernelInfo BuildSYCLKernelInfo(ASTContext &Context,
```

Comment:
`Context` parameter is unused. I suppose it might trigger a compiler warning. Can we drop this parameter?

Reply:
Oops, yes. It will be needed in future PRs, but should be added then. I'll remove it in this one.

Reply:
Done.